ad/models.py

```diff
@@ -0,0 +1,177 @@
+from django.db import models
+from django.contrib.auth.models import User
+import datetime
+
+class AdInfo(models.Model):
+    title = models.CharField(
+        max_length=255,
+    )
+    description = models.TextField()
+    tags = models.CharField(
+        max_length=255,
+        blank=True,
+    )
```

Comment:
You can use django_taggit package for tags. It will be a good practice for you because we are using this package in testpress codebase.
https://pypi.org/project/django-taggit/

Reply:
ok